test/implement_pitch_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class TestNote < MiniTest::Test
+  def setup
+    @pitch_object_note = MusicTheory::Note.new(frequency: Pitch.new(:B, 4), duration: 1.0, distort: false)
+    @frequency_note    = MusicTheory::Note.new(frequency: 493.883, duration: 1.0, distort: false)
```

Comment:
a pitch is not a frequency, which confuses this interface a bit. perhaps `pitch: Pitch.new(:B, 4)` instead